<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oknoARX</class>
 <widget class="QDialog" name="oknoARX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model ARX</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="aLineEdit">
         <property name="text">
          <string>-0.4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bLineEdit">
         <property name="text">
          <string>0.6</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Zaklocenie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="zaklocenieSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Opoznienie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="opoznienieSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="zatwierdzButton">
         <property name="text">
          <string>ZATWIERDZ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="anulujButton">
         <property name="text">
          <string>ANULUJ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
